deltacat/tests/compute/compactor_v2/utils/test_exception_handler.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from deltacat.compute.compactor_v2.utils.exception_handler import (
+    handle_exception,
+)
+
+from deltacat.exceptions import (
+    DependencyPyarrowCapacityError,
+)
+from pyarrow.lib import ArrowCapacityError
+
+
+class TestExceptionHandler(unittest.TestCase):
+    @handle_exception
+    def mock_raise_exception(self, exception_to_raise):
+        raise exception_to_raise
+
+    def test_exception_handler(self):
```

Comment:
We need one test all top level branches atleast preferably the ones where we extract the wrapped exception and re-raise the error.